test/utils/config/goerli.json

```diff
@@ -14,7 +14,7 @@
   "UNISWAP_V3_ADDRESS": "0xE592427A0AEce92De3Edee1F18E0157C05861564",
   "UNISWAP_V3_QUOTER_ADDRESS": "0x0000000000000000000000000000000000000000",
   "UNISWAP_PERMIT2_ADDRESS": "0x000000000022d473030f116ddee9f6b43ac78ba3",
-  "UNISWAP_UNIVERSAL_ROUTER_ADDRESS": "0x4648a43B2C14Da09FdF82B161150d3F634f40491",
+  "UNISWAP_UNIVERSAL_ROUTER_ADDRESS": "0x3fC91A3afd70395Cd496C647d5a6CC9D4B2b7FAD",
```

Comment:
Also add swapRouter02 address into the config?

Reply:
Yeah, added f01552c